apps/sms/js/thread_list_ui.js

```diff
@@ -402,10 +403,18 @@ var ThreadListUI = {
         thread.unreadCount = isRead ? 0 : 1;
         this.mark(thread.id, isRead ? 'read' : 'unread');
```

Comment:
For `this.mark` I think you're right, but I'm not sure for thread.unreadCount. OK maybe you're right for now but when we'll have a DB in Gaia maybe we'll need something more subtle.